Rust 2019 5,6,7,8,9 (#12)
skade/yakshav.es: _sources/rust-2019-beyond-rustconf-and-rustfest.md

---
title: "Rust 2019 (10): Beyond RustConf and RustFest"
---

# Rust 2019 (10): Beyond RustConf and RustFest

When it comes to events, Rust has had it good: all larger conferences are currently organised by a ton of experienced conference organisers. RustConf by Tilde of RailsConf and EmberConf fame, RustFest by people behind multiple European community confs, Rust Belt Rust by Carol of Steel City Ruby fame! I can only speak for RustFest, but we were driven by the need to avoid classic community beginner mistakes, especially reinventing wheels around standards of diversity, accessibility and inclusion.

I'm beginning to wonder if we hadn't set the standard to high a little in the process: RustFest is now a _beast_. It is three different events in one (Conference, Workshops and hackfest), all with different needs to organise, every 6 months. Even considering that RustFest implementation quality could be improved, it's _hard_ to live up to. That's fine, RustFest was never intended to be a benchmark for all.

> Whenever someone says "I want it to be like RustFest", it makes me shiver. RustFest is the event of the RustFest team. With a very specific style, a certain acceptance for its flaws but a lot of love for some details. It is a community event, but it is also _our_ event. I don't mean this defensively - it just means that many of our preferences prefer some solutions over the other. I personally want to empower people to run _their_ events. That also means that they should not have false respect to whats there. You think a straight-forward two-day conference is what's lacking in the european Rust space? You're probably right, go ahead, do it! You don't like RustFests style? By all means, don't adopt it. There's tons of ways to organise and event and I would love to see people get more creative. Wa

Reviewer: ```suggestion
Whenever someone says "I want it to be like RustFest", it makes me shiver. RustFest is the event of the RustFest team. With a very specific style, a certain acceptance for its flaws but a lot of love for some details. It is a community event, but it is also _our_ event. I don't mean this defensively - it just means that many we prefer some solutions over the other. I personally want to empower people to run _their_ events. That also means that they should not have false respect to what's there. You think a straight-forward two-day conference is what's lacking in the European Rust space? You're probably right, go ahead, do it! You don't like RustFest's style? By all means, don't adopt it. There's tons of ways to organise an event and I would love to see people get more creative.
```

Reviewer: hmmm, I hope above gets rendered properly. I fixed a few small typos only.

> I hope that in 2019, a couple of independent users will decide to run their own thing. In Europe, the US, and beyond. RustRush and Rust LATAM are showing a good path!

Reviewer: `users` sounds weird in this context. Maybe just people/folks or community members?

Reviewer: Maybe some links are appropriate to these confs.
